miniupnpc/miniupnpc.c

```diff
@@ -432,7 +432,7 @@ build_absolute_url(const char * baseurl, const char * descURL,
 	base = (baseurl[0] == '\0') ? descURL : baseurl;
 	n = strlen(base);
 	if(n > 7) {
-		p = strchr(base + 7, '/');
+		p = (char *)strchr(base + 7, '/');
```

Comment:
Now I recalled that this cast came from VS2012, where `strchr()` was returning `const char *`.